components/generic/CompareImageSlider.stories.js

```diff
@@ -0,0 +1,21 @@
+import { storiesOf } from '@storybook/vue';
+
+import CompareImageSlider from './CompareImageSlider.vue';
+
+storiesOf('Generic', module)
+  .add('Compare Image Slider', () => ({
+    components: { CompareImageSlider },
+    data() {
+      return {
+        imageLeft: 'https://www.fillmurray.com/640/360',
```

Comment:
Would we compare images like this? @rwd thoughts?

Reply:
This is more about the fact that we want to avoid using external dependencies.
However using a landscape vs. portrait comparison is also a good example to show how we will handle this case.